src/ecrc-frontend/src/components/base/button/Button.test.js

```diff
@@ -21,4 +21,21 @@ describe("Button Component", () => {
     );
     expect(button.toJSON()).toMatchSnapshot();
   });
+  test("Unknown button type", () => {
```

Comment:
what is this test trying to assert?

Reply:
there is a bit of logic in the button component to check for valid button types. This tests a part of that conditional branching.